<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задача</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/Без названия.png</normaloff>../img/Без названия.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #333333;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2" colspan="3">
      <widget class="QPushButton" name="taskColorButton">
       <property name="styleSheet">
        <string notr="true">background-color: #777777;color: #ffffff;
</string>
       </property>
       <property name="text">
        <string>Выбрать цвет</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="widgetColorText" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: #ffffff;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;</string>
       </property>
       <property name="text">
        <string>Цвет задачи</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;</string>
       </property>
       <property name="text">
        <string>Цвет текста</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;</string>
       </property>
       <property name="text">
        <string>Заголовок</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="5">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="styleSheet">
        <string notr="true">background-color: #777777;color: #ffffff;
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;</string>
       </property>
       <property name="text">
        <string>Описание</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QLabel" name="err_msg">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QPushButton" name="textColorButton">
       <property name="styleSheet">
        <string notr="true">background-color: #777777;
color: #ffffff;
</string>
       </property>
       <property name="text">
        <string>Выбрать цвет</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="header">
       <property name="styleSheet">
        <string notr="true">background-color: #555555;
color: #ffffff;
border: none;</string>
       </property>
       <property name="text">
        <string>Новая задача</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QPlainTextEdit" name="description">
       <property name="styleSheet">
        <string notr="true">background-color: #555555;
color: #ffffff;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QWidget" name="widgetColorTask" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: #666666;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;</string>
       </property>
       <property name="text">
        <string>Группа</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="todoButton">
         <property name="styleSheet">
          <string notr="true">color: #ffffff;</string>
         </property>
         <property name="text">
          <string>To do</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="progressButton">
         <property name="styleSheet">
          <string notr="true">color: #ffffff;</string>
         </property>
         <property name="text">
          <string>Progress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="doneButton">
         <property name="styleSheet">
          <string notr="true">color: #ffffff;</string>
         </property>
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
